prover/src/trace/poly_table.rs

```diff
@@ -102,8 +107,8 @@ impl<E: FieldElement> TracePolyTable<E> {
         self.main_segment_polys.columns()
     }
 
-    /// Returns an iterator over the polynomials of all auxiliary trace segments.
-    pub fn aux_trace_polys(&self) -> MultiColumnIter<E> {
+    /// Returns an iterator over the polynomials of the auxiliary trace segment.
+    pub fn aux_segment_polys(&self) -> MultiColumnIter<E> {
         MultiColumnIter::new(self.aux_segment_polys.as_slice())
     }
```

Comment:
We can probably replace `MultiColumnIter` with just `ColumnIter` and maybe delete `MultiColumnIter` altogether.